kl_participants/migrations/0001_initial.py

```diff
@@ -17,6 +17,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('display_name', models.CharField(max_length=255)),
+                ('region', models.CharField(choices=[('dolnoslaskie', 'dolnoslaskie'), ('slaskie', 'slaskie')], max_length=32)),
```

Comment:
Not all regions.

Reply:
Look at ticket with initial handshake - it should explain why only hostname will be enough :)

Reply:
Additionally, we will have control over all those machines